### ISSUE TITLE
How does paths option really work?

### ISSUE THREAD

**Issue by robhrt7** [terrierscript/grunt-cssjoin#4]
I have this kind of CSS file:

```
@import "/res/css/reset.css";
@import "/res/css/base/base.css";
```

and Grunt config:

```
cssjoin: {
    bundles: {
        options: {
            paths: ['build/grunt']
        },
        files: [
            {
                expand: true,
                cwd: 'path/with/import-bundles',
                dest: 'comp/some',
                src: [
                    '**/*.css'
                ]
            }
        ]
    }
}
```

In cases, where defined file path in `@import` could not be found (on filesystem), task uses `options.paths.path` + `declared import path`, but if file that is declared in `@import` statement exists, options.paths are not used in forming right path to file that I need to join.

So if `/res/css/reset.css` exists, cssjoin will take file from there, but if not, if would combine path in options.paths and will lookup file in this dir `build/grunt/res/css/reset.css`. How could I make cssjoin to only look for path that are combined with option defined rule?

**Comment by robhrt7** [comment]
Seems like it's cssjoin issue, I have made an issue there either https://github.com/suisho/cssjoin/issues/10

**Comment by robhrt7** [comment]
Hm, I have found that grunt-cssjoin is using year old cssmin, but the newest version of cssmin is enhanced with image url replacement, that isn't described in this task. Does grunt-cssjoin will be using newest versions of cssjoin? Will be there an option to turn off image url replacement?

**Comment by suisho** [comment]
Ok, I upgrade cssjoin. Thank you!
(But cssjoin not depend cssmin. typo?)

**Comment by robhrt7** [comment]
Yeah, sorry, typo. So you'll update cssjoin 0.2.0 version? Because now cssjoin lib does not only the @imports concatenation, but also changes image url, what average grunt-cssjoin users wont expect.
